codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandshakeHandler.java

```diff
@@ -125,4 +155,44 @@ private static String getWebSocketLocation(ChannelPipeline cp, HttpRequest req,
         String host = req.headers().get(HttpHeaderNames.HOST);
         return protocol + "://" + host + path;
     }
+
+    public WebSocketServerProtocolHandshakeHandler handshakeTimeoutMillis(long handshakeTimeoutMillis) {
```

Comment:
package-private as the class is package-private as well

Reply:
Done